### DISCUSSION TITLE
How to use postman as client to call workflow

### DISCUSSION THREAD

**Discussion by volethanh** [run-llama/llama_deploy#350] [Q&A]
Hi, 

As the document has only mentioned about using llamaClient to call workflow or service, I just want to know how to use Postman to call API for specific workflow, which url link (control plan or direct url of workflow ?) and what body of that request.

**Comment by logan-markewich** [comment]
Launch the control plane and then your workflow

Then checkout the control plane swagger docs at  `http://127.0.0.1:8000/docs` (the default url for the control plane, if not modified)

If you look at the client code, you first create a session, and then use that session ID to run
https://github.com/run-llama/llama_deploy/blob/4862036f8bbe26bf6e69a842d56406698620ac47/llama_deploy/client/sync_client.py#L126

Then once you have a session ID, run it
https://github.com/run-llama/llama_deploy/blob/4862036f8bbe26bf6e69a842d56406698620ac47/llama_deploy/client/sync_client.py#L32
